options.go

```diff
@@ -280,6 +283,24 @@ func TombstonePeriod(period time.Duration) Option {
 	}
 }
 
+// SelfEvictPingDisable configures if ringpop should disable actively pinging
+// other members when it evicts itself from the network
+func SelfEvictPingDisable(disable bool) Option {
+	return func(r *Ringpop) error {
+		r.config.SelfEvict.PingDisable = disable
+		return nil
+	}
+}
+
+// SelfEvictPingRatio configures the maximum percentage/ratio of the members to
+// actively ping while self evicting.
```

Comment:
I think we could improve the documentation here a bit by adding some more information about why you would want to tune this.

E.g.:

`// A higher number means xyz where as a lower number means xyz`

In other words, explain the use of this and the trade-off.